indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/BERTags.kt

```diff
@@ -47,29 +50,30 @@ object BERTags {
     const val CONSTRUCTED: UByte = 0x20u // decimal 32
     const val UNIVERSAL: UByte = 0x00u // decimal 32
     const val APPLICATION: UByte = 0x40u // decimal 64
-    const val TAGGED: UByte = 0x80u // decimal 128 - maybe should deprecate this.
     const val CONTEXT_SPECIFIC: UByte = 0x80u // decimal 128
     const val PRIVATE: UByte = 0xC0u // decimal 192
     const val FLAGS: UByte = 0xE0u
 
 }
 
-object DERTags {
-    val DER_SEQUENCE: UByte = BERTags.CONSTRUCTED or BERTags.SEQUENCE
-    val DER_SET: UByte = BERTags.CONSTRUCTED or BERTags.SET
-    fun UByte.toExplicitTag() = BERTags.CONSTRUCTED or BERTags.TAGGED or this
-
-    val UByte.isExplicitTag get() = ((this and BERTags.CONSTRUCTED) != 0.toUByte()) && ((this and BERTags.TAGGED) != 0.toUByte())
+/**
+ * Convenience helper to easily construct implicitly tagged elements. Can be CONSTRUCTED or PRIMITIVE
+ * @param constructed whether to set the constructed bit
+ */
+fun ULong.toImplicitTag(constructed: Boolean = false) =
```

Comment:
should this be `internal`/`private` if it's a "convenience helper"? also maybe `asAsn1Tag` or something?